src/com/lftechnology/java/training/srijan/basics/javaio1/IntegerReader.java

```diff
@@ -0,0 +1,58 @@
+package com.lftechnology.java.training.srijan.basics.javaio1;
+
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.logging.Logger;
+
+/**
+ * A file which has some int value among with other string value . Write a
+ * program that gets the int piece of data. What is the int data?
+ * 
+ * @author srijan
+ *
+ */
+public class IntegerReader {
+
+	private static final Logger LOGGER = Logger.getLogger(IntegerReader.class.getName());
+
+	private IntegerReader() {
+
+	}
+
+	public static void main(String[] args) {
+		readIntegerFromFile();
+	}
+
+	/**
+	 * <p>
+	 * Reads Integer from file inputfile.txt
+	 * </p>
+	 * <p>
+	 * Displays integer present in file
+	 * </p>
+	 * 
+	 * @author srijan
+	 */
+	public static void readIntegerFromFile() {
+		try {
+			FileReader readFromFile = new FileReader("inputfile.txt");
+			int integerInFile = 0;
+			try {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Extract this nested try block into a separate method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1141)